lldb/include/lldb/Utility/RegisterValue.h

```diff
@@ -69,7 +70,7 @@ class RegisterValue {
     m_scalar = inst;
   }
 
-  explicit RegisterValue(llvm::APInt inst) : m_type(eTypeUInt128) {
+  explicit RegisterValue(llvm::APInt inst) : m_type(eTypeUInt) {
```

Comment:
These should be eTypeUintN now, right?

(this and the 2 below in this same file)